### ISSUE TITLE
strange Blurred an black brushed coloring on Y axis numbers

### ISSUE THREAD

**Issue by ubersant** [oxyplot/oxyplot#1866]
### Steps to reproduce

![image](https://user-images.githubusercontent.com/11734961/157332986-a602dc02-a21d-46a5-aa4f-7711924d7179.png)

snippet 

![image](https://user-images.githubusercontent.com/11734961/157332871-b0738637-7a76-4dac-bf88-3ff8f36a8a55.png)

1. Been having two plots, one plots just fine and other is appearing as shown in image above
2. below is the code block
3. do add only one value each for x and y respectively 

Platform:
.NET version:

### Expected behaviour

Tell us what should happen

### Actual behaviour

Tell us what happens instead
Can you also include a screen shot?

### Feature description

Write a description of the feature. How should it work? How should it look?
Include some graphics if this could help!

**Comment by VisualMelon** [comment]
It's just rendering way too many axis ticks: setting `MajorTick` and `MinorTick` explicitly overrides the internal logic for computing tick distributions, and plots them at the given intervals instead.

**Comment by ubersant** [comment]
Okay so I have to be explicitly setting up majortick and minortick in that snippet ? 

And second, can you please point me to a document or some reference for this please.

Thanks for your suggestions @VisualMelon 

Thanks

**Comment by VisualMelon** [comment]
@ubersant sorry, I meant `MajorStep` and `MinorStep`: because you are specifying them they are overriding the normal logic which picks a distribution based on the visible range and the (screen space) `IntervalLength` .

I can't find any documentation of this off hand; sadly the state of OxyPlot's documentation has never been fantastic, the most up-to-date documentation is at https://github.com/oxyplot/docs. If I have the time I'll have a go at updating the axis documentation over the weekend. This sort of information should ideally be specified in the inline documentation also (the `ActualMinorStep` is computed from ... if the `MinorStep` is `NaN`, otherwise it is `MinorStep`, clamped to `MinimumMinorStep` etc.)

**Comment by ubersant** [comment]
Okay and now when I remove the major and minor from my code above, it gets rid off that thick black line but then lines aren't plotted in graph, could be a separate issue ? 

Two lines in that graph have only one set of coordinates, is it the reason maybe ? Two set would have made it a line probably

**Comment by VisualMelon** [comment]
Yes, a line series expects many data points. If you set a `MarkerStyle` (e.g. `Circle`) then you should see those points plotted. If you want to show an upper/lower bound, then consider a `LineAnnotation`.

**Comment by ubersant** [comment]
how about this @VisualMelon ? If i wanted the upper line to continue ? I have colored it as yellow for presentation purpose 

![yellow](https://user-images.githubusercontent.com/11734961/157436426-3b708699-eab0-4e8c-8da2-239d17479f2a.png)

**Comment by VisualMelon** [comment]
I've discussed this sort of thing with people before, and I would assert is that it doesn't make sense to do this on a series with real data.

**Comment by ubersant** [comment]
oh okay, so it would not be possible to do you mean ? @VisualMelon

**Comment by VisualMelon** [comment]
It's possible: I implemented something similar as part of a discussion (here or on the gitter, can't find it right now), but I don't think it makes sense in a general-purpose series (i.e. your data should have some sort of time-limit, the line shouldn't extend to infinitely because that's probably not a reasonable interpolation (though could be in your specific case)). It's fairly simple to add a 'virtual' point to the line series that extends to the edge of the plot (which is what I think your picture is representing?) as `LineAnnotation` does.

**Comment by ubersant** [comment]
@VisualMelon okay got your point, how about adding circle dots to the points ? so that one understands where points are as line extends from one side to other end...helps recognizing ...any idea on how one can add dot over those points ?

**Comment by VisualMelon** [comment]
Sounds like you want `LineSeries.MarkerType = MarkerType.Circle`

**Comment by ubersant** [comment]
yes perfect that, thanks much @VisualMelon 

coming to the extended lines, is it the one you referring to ? https://github.com/oxyplot/oxyplot/issues/1750

I am unable to find any 'virtual' property inside series, just thinking how to plot endless lines

**Comment by VisualMelon** [comment]
Yeah that's the one whereabout I was thinking. The code example (https://gist.github.com/VisualMelon/ea288e012923e5e94cb27c9001c35536#file-signalseries-cs-L21-L43) shows how you can easily extend line series with this sort of functionality (though it is a hideous bodge): you just need to change `EndX` to whatever you want (e.g. you can ask the `XAxis` what it's extent is and use that.

**Comment by ubersant** [comment]
oh okay, so I need to add this custom code. i thought it would be like just mentioning some property in the lineseries object haha

you have helped me a lot all the while, thanks a ton, lastly do you see any other alternatives for implementing endless lines? 

How do i buy you a coffee ? :)

**Comment by VisualMelon** [comment]
Yeah, it's not an existing feature.

You could implement the feature at various levels; it would really depend on what API you want.

**Comment by VisualMelon** [comment]
I have added some more documentation about axes here: https://github.com/oxyplot/docs/pull/49

Feel free to provide feedback if its useful or unclear.
